src/actions/auth/types.ts

```diff
@@ -0,0 +1,16 @@
+import { AuthStatus } from './dto'
+
+export type AuthActionProps = {
+	request: Request
+}
+
+export type AuthActionData = {
+	status: null | AuthStatus
+	message?: string
+	user?: { email: string; password?: string }
+}
+
+export type LoginError = {
```

Comment:
looks like a generic type for Erros to me. so you can just rename it and move it to the generic types